main.go

```diff
@@ -1,25 +1,72 @@
 package main
 
 import (
-    "notes-service/grpc/notespb"
-    "net"
-    
-    "fmt"
+	"context"
+	"fmt"
+	"net"
+	"time"
+
+	"notes-service/grpc/notespb"
+
+	"google.golang.org/grpc"
+
+	"go.mongodb.org/mongo-driver/mongo"
+	option "go.mongodb.org/mongo-driver/mongo/options"
+	"go.mongodb.org/mongo-driver/mongo/readpref"
+
+	log "github.com/sirupsen/logrus"
 )
 
+var NotesDatabase *mongo.Database = nil
+var NotesCollection *mongo.Collection = nil
+
+// Need to put these variables in .env
+var password = "Gyy628\\nAWS"
+var mongoUri string = "mongodb+srv://gabriel:" + password + "@cluster0.2ckb3.mongodb.net/myFirstDatabase?retryWrites=true&w=majority"
+
+func InitDatabase() {
+	client, err := mongo.NewClient(option.Client().ApplyURI(mongoUri))
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+
+	err = client.Connect(ctx)
+	fmt.Println("service is connected to MongoDB")
+
+	if err != nil {
+		defer client.Disconnect(ctx)
+		log.Fatal(err)
+	}
+
+	NotesDatabase = client.Database("Notes-database")
+	NotesCollection = NotesDatabase.Collection("Notes-collection")
+
+	err = client.Ping(ctx, readpref.Primary())
+	if err != nil {
+		defer client.Disconnect(ctx)
+		log.Fatal(err)
+	}
+}
+
 func main() {
-    srv := grpc.NewServer()
-    notesSrv := notesService{}
-    notespb.RegisterAccountsServiceServer(srv, &notesSrv)
 
-    lis, err := net.Listen("tcp", ":3000")
-    
-    ftm.Println("server listen on port 3000")
+	InitDatabase()
+
+	srv := grpc.NewServer()
+	notesSrv := notesService{}
+	notespb.RegisterNotesServiceServer(srv, &notesSrv)
+
+	lis, err := net.Listen("tcp", ":3000")
+
+	fmt.Println("service listen on port 3000")
```

Comment:
Sometimes you use the `fmt` package for logging and other times `log`. `fmt` is not appropriate for logging. Also we've decided to use structured logging for the Noted backend so I think you should use `uber-go/zap` for all logging.

Reply:
You can draw inspiration from the accounts-service, everything's already been implemented and reviewed there in terms of logging.